geo/src/algorithm/bool_ops/mod.rs

```diff
@@ -140,3 +224,27 @@ impl<T: BoolOpsNum> BooleanOps for MultiPolygon<T> {
         self.0.iter().flat_map(|p| p.rings())
     }
 }
+
+/// Allows the unary union operation to be performed on any container which can produce items of type `Polygon<T>`
+impl<T, Container> UnaryUnion for Container
+where
+    T: BoolOpsNum,
+    Container: IntoIterator<Item = Polygon<T>> + Clone,
+    Polygon<T>: RTreeObject,
+{
+    type Scalar = T;
+
+    fn unary_union(&self) -> MultiPolygon<Self::Scalar> {
+        // these three functions drive the union operation
+        let init = || MultiPolygon::<T>::new(vec![]);
```

Comment:
This looks like `MultiPolygon` might benefit from a `Default` impl so this could be written as

```suggestion
        let init = MultiPolygon::<T>::default;
```

Reply:
I think the same goes for all our multi-types, but probably better to address that outside of this PR.